docs/datamodel/objects.rst

```diff
@@ -8,9 +8,11 @@ Object Types
 analogous to SQL *tables* or ORM *models*, and consist of :ref:`properties
 <ref_datamodel_props>` and :ref:`links <ref_datamodel_links>`.
 
-Properties are used to attach primitive data to an object type. They are
-declared with the ``property`` keyword. For the full documentation on
-properties, see :ref:`Properties <ref_datamodel_props>`.
+Properties are used to attach primitive data to an object type. Prior to
+EdgeDB 3.0, they must be declared with the ``property`` keyword, but the
+keyword is not required in 3.0+ unless the property is a :ref:`computed
+property <ref_datamodel_computed>`. For the full documentation on properties,
+see :ref:`Properties <ref_datamodel_props>`.
```

Comment:
I thought it might be good to remind the reader why the `property` and `link` keywords are used here, since when I first read this section, I thought they were typos left over from the pre-3.0 syntax. However, let me know if you have any suggestions for how this could be worded better, or if you don't think this extra info is needed.

Reply:
Looks good and the extra reminder makes sense. I'd make a small change to this because as is it feels like "must" should be in the past tense:

"In versions prior to EdgeDB 3.0 they must be declared with the..."

Reply:
I agree, that would read better. I've updated it to use your suggestion.